photutils/mask_data.py

```diff
@@ -0,0 +1,125 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Functions for creating a cut out of the data and a mask of the same cut out
+corresponding to the aperture"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+import warnings
+import astropy.units as u
+from astropy.utils.exceptions import AstropyUserWarning
+
+
+__all__ = ['get_cutouts']
+
+
+
+def get_cutouts(data, circular_aperture, use_exact = 0, subpixels = 5):
+    '''
+    Create a square cutout of each aperture and an associated mask which denotes the
+    aperture with 1 for inside, 0 for outside, and a fraction for the edges.
+
+    Parameters:
+    -----------
+    data : Image array
+    circular_aperture: photutils aperture object
+    use_exact : (default = 0) refers to the keyword in .geometry.circular_overlap_grid.
```

Comment:
add a line brake here, so the first one only contains the name of the parameter and the type